api/apps/api/src/utils/__mocks__/fake-logger.ts

```diff
@@ -1,4 +1,8 @@
-export class FakeLogger {
+import { LoggerService } from '@nestjs/common';
+
+export class FakeLogger implements LoggerService {
+  constructor(public readonly context?: string) {}
+
```

Comment:
why is this context needed?

Reply:
Thanks, it indeed isn't needed https://github.com/Vizzuality/marxan-cloud/pull/417/commits/19df1e8698cecb125b237f3ca179ed1b846121ca